.npmignore

```diff
@@ -0,0 +1,10 @@
+# blacklist all
+**
+
+# include whitelist
+!dist/*
+!dist/components/*
+!LICENSE
+!ACKNOWLEDGMENTS
+!README.md
+!package.json
```

Comment:
While `package.json` is one of the files that are always included, as are `README.md` and `LICENSE`, I figure having them typed out here only helps with clarity.

Reply:
Hi @rschristian. Thanks.
I on a vacation: will test as soon as I have time.

Have you run `npm publish --dry-run`?

Reply:
No problem,enjoy your vacation. Nothing pressing about this. 

Yes, I have tested with `publish --dry-run`. That's where I got the file count from.